src/harness/security_testcase.py

```diff
@@ -66,6 +67,14 @@ def SasReset(self):
     """Resets the SAS UUT to its initial state."""
     self._sas_admin.Reset()
 
+  def getCertFilename(self, cert_name):
```

Comment:
This appears unrelated to the rest of this PR.

Reply:
removed this change